index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <link rel="stylesheet" href="styles.css" />
+        <link
+            rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+        />
+    </head>
+
+    <body class="container">
+        <header class="header">
```

Comment:
Great design @Developer-Kingz 🤝

- [ ] As stated in the project requirements, we should stick to the design guidelines as much as possible. Kindly ensure your design consists of the first Navbar just like it appeared in the design provided.
- Your design
![Your Navbar](https://github.com/Developer-Kingz/Microverse-capstone/assets/63917725/0e178f99-b1ce-438e-ac6f-17bf629d4fbe)
- Given design
![Given Navbar](https://github.com/Developer-Kingz/Microverse-capstone/assets/63917725/09689d2d-9f03-4525-9434-c27b43c2dea0)